src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets

```
@@ -48,6 +48,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <RuntimeIdentifiers Condition=" '$(PackAsTool)' == 'true' ">$(RuntimeIdentifiers);$(PackAsToolShimRuntimeIdentifiers)</RuntimeIdentifiers>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(EnablePreviewFeatures)' == 'true'">
```

Is it possible to only set `LangVersion` here if it's not already explicitly set, or does this happen after the default is already set in the targets?

```suggestion
  <PropertyGroup Condition="'$(EnablePreviewFeatures)' == 'true' And '$(LangVersion)' == ''">
```

---

Ah you're right, we can set it only when `LangVersion` has not been set explicitly. Fixed now.